problems/getCountriesSortedByPopulation.js

```diff
@@ -10,8 +10,36 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
-
+// takes in an array with objects holding a country as key and it's population number as value
+// need to return country names with lowest to highest population
+// will sort population number from lowest to highest
+// will access the property values with arr.country
+// 
+function getCountriesSortedByPopulation(arr) {
+    let newObj = {}
+    for (let element in arr) {
+        newObj[element] = arr[element]
```

Comment:
Consider what 'element' is, which information you need from the element, and which array method allows you to add data into the array you are going to return.